features/main/always_identifier.feature

```diff
@@ -0,0 +1,117 @@
+Feature: AlwaysIdentifier support
```

Comment:
May  I suggest: `Feature: allow resources in properties to always be serialized as identifiers`